src/ansys/mechanical/core/embedding/app.py

```diff
@@ -71,6 +74,19 @@ def __init__(self, db_file=None, **kwargs):
         if self._version == None:
             self._version = _get_default_version()
         initializer.initialize(self._version)
+
+        pkgs = freeze.freeze()
+        for pkg in pkgs:
+            indx = pkg.index("=")
+            if pkg[:indx] == "pythonnet":
+                warnings.warn(
+                    "The pythonnet package was found in your environment"
+                    "which interferes with the ansys-pythonnet package. "
+                    "Some APIs may not work due to pythonnet being installed.",
+                    stacklevel=3,
+                )
+                break
```

Comment:
Nice implementation @klmcadams =) could you unit test this feature if possible?

Reply:
Thanks! Sure, I will add them in soon.